org.scala-refactoring.library/src/main/scala/scala/tools/refactoring/common/tracing.scala

```diff
@@ -21,7 +26,32 @@ trait Tracing {
   def trace(msg: => String)
 }
 
-trait ConsoleTracing extends Tracing {
+object DebugTracing {
+  private val debugStream = {
+    Option(System.getProperty("scala.refactoring.traceFile")).flatMap { fn =>
```

Comment:
Such property names should be mentioned separately somewhere, otherwise we can't keep an overview over them. You could keep them in an separate object definition somewhere.